packages/puppeteer-core/src/api/ElementHandle.ts

```diff
@@ -885,7 +895,63 @@ export class ElementHandle<
    * or `null` if the element is not visible.
    */
   async boundingBox(): Promise<BoundingBox | null> {
-    throw new Error('Not implemented');
+    const adoptedThis = await this.frame.isolatedRealm().adoptHandle(this);
+    const box = await this.frame.isolatedRealm().evaluate(element => {
```

Comment:
Can this be simplified to
```suggestion
    const box = await adoptedThis.evaluate(element => {
```
?